src/components/Tool/Appeal/Modals/AddContactToAppealModal/appealInfo.graphql

```diff
@@ -0,0 +1,6 @@
+query Appeal($accountListId: ID!, $appealId: ID!) {
```

Comment:
Do you mind changing the case of this file too?

Reply:
I did that on the previous PR, which will be merged into this one., but I will do it here as well.

Reply:
You can do it later too. Either way is fine.